sign/dilithium/internal/common/field_test.go

```diff
@@ -4,22 +4,25 @@ import (
 	"crypto/rand"
 	"encoding/binary"
 	"flag"
+	"math"
 	"testing"
 )
 
 var runVeryLongTest = flag.Bool("very-long", false, "runs very long tests")
 
-func randSliceUint32(N uint) []uint32 {
-	bytes := make([]uint8, 4*N)
+func randSliceUint32(length uint) []uint32 { return randSliceUint32Max(length, math.MaxUint32) }
+
+func randSliceUint32Max(length uint, max uint32) []uint32 {
```

Comment:
I would suggest something like `randSliceUint32WithMax` to make it clear that this function has an *additional* parameter called `max`.

Reply:
done